sources/platform/storage/request_queue.md

```diff
@@ -457,23 +467,71 @@ await requestQueueClientOne.batchAddRequests([
 const processingRequestsClientOne = await requestQueueClientOne.listAndLockHead(
     {
         limit: 2,
-        lockSecs: 60,
+        lockSecs: 120,
     },
 );
 
+// Checks when the lock will expire. The locked request will have a lockExpiresAt attribute.
+const theFirstRequestLockedByClientOne = processingRequestsClientOne.items[0];
+const requestLockedByClientOne = await requestQueueClientOne.getRequest(
```

Comment:
The naming of these two variables is pretty confusing, I don't really know what's the difference between `theFirstRequestLockedByClientOne` and `requestLockedByClientOne`, especially by name. Could you come up with some better names?